TOPICS.md

```diff
@@ -74,73 +96,79 @@ For UIs that show more information, so also voltage and current, its a bit more
+```
+N/{portalId}/system/0/Ac/Grid/*                      <- All from the shore. TODO: check if this shouldn't be /Ac/Shore
+N/{portalId}/system/0/Ac/Genset/*                    <- All from the generator.
+```
 
-Basically, what you want to find out is the instance number of the vebus service. These are
-the available topics on system/0 that might help to do that:
+#### Inverter/charger data:
 
 ```
-ServiceMapping/com_victronenergy_charger_0                com.victronenergy.charger.socketcan_can0_di0_uc15639
-ServiceMapping/com_victronenergy_settings_0                                         com.victronenergy.settings
-ServiceMapping/com_victronenergy_solarcharger_0      com.victronenergy.solarcharger.socketcan_can0_di0_uc15965
-ServiceMapping/com_victronenergy_solarcharger_258                         com.victronenergy.solarcharger.ttyO2
-ServiceMapping/com_victronenergy_vebus_257                                       com.victronenergy.vebus.ttyO1
-ServiceMapping/com_victronenergy_vecan_0                                          com.victronenergy.vecan.can0
-VebusService                                                                     com.victronenergy.vebus.ttyO1
+N/{portalId}/system/0/Dc/Vebus/Current               <- charge/discharge current between battery
+                                                    and inverter/charger
+
+N/{portalId}/system/0/Dc/Vebus/Power                 <- same, but then the power
+
+N/{portalId}/system/0/SystemState/State              <- Absorption, float, etc.
+                                                       0 - "Off" 1 - "Low power"
+                                                       2 - "VE.Bus Fault condition"
+                                                       3 - "Bulk charging"
+                                                       4 - "Absorption charging"
+                                                       5 - "Float charging"
+                                                       6 - "Storage mode"
+                                                       7 - "Equalisation charging"
+                                                       8 - "Passthru"
+                                                       9 - "Inverting"
+                                                       10 - "Assisting"
+                                                       256 - "Discharging"
+                                                       257 - "Sustain"
+
+N/{portalId}/system/0/Ac/ActiveIn/Source             <- The active AC-In source of the multi.
+                                                       0:not available, 1:grid, 2:generator,
+                                                       3:shore, 240: inverting/island mode.
+
+N/{portalId}/system/0/Ac/Consumption/NumberOfPhases  <- Either 1 (single phase), 2 (split-phase) or
+                                                        3 (three-phase)
+
+N/{portalId}/system/0/Ac/ConsumptionOnOutput/*       <- Use this for AC Consumption.
```

Comment:
Hey @ucorina, is this correct? I thought you now take this data from .vebus; the reason being that via .system you can only get the powers. Not voltage and current.

Reply:
We take the power from .system, and the voltage and current from .vebus. Maybe I should change this to /Power to make it more clear. See also https://github.com/victronenergy/venus-html5-app/pull/84/files#diff-c7b0a051f14a126f68065616ad894f83R229 a bit lower in the file, for the exact paths used in the AC Loads box.
I actually had a note from our initial discussions to not use `N/{portalId]/vebus/{vebusInstanceId}/Ac/Out/L1/P` for getting the AC consumption.

Reply:
Hi; yes; thats probably from  back in the days that I thought you could get all you needed from system.

Since you now need power, voltage and current, I recommend to take them all from vebus. Not only power from .system. To make it more simple.

Reply:
Ok, just updated the code and the docs.